guides/common/modules/ref_system-requirements.adoc

```diff
@@ -43,14 +42,16 @@ endif::[]
 
 {Project} only supports `UTF-8` encoding.
 If your territory is USA and your language is English, set `en_US.utf-8` as the system-wide locale settings.
-For more information about configuring system locale in {EL}, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/assembly_changing-basic-environment-settings_configuring-basic-system-settings#proc_configuring-the-system-locale_assembly_changing-basic-environment-settings[Configuring System Locale guide].
+ifdef::foreman-el,katello,satellite[]
+For more information about configuring system locale in {EL}, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/assembly_changing-basic-environment-settings_configuring-basic-system-settings#proc_configuring-the-system-locale_assembly_changing-basic-environment-settings[Configuring the system locale] in _{RHEL} 8 {Configuring basic system settings}_.
+endif::[]
 
 {ProjectServer} and {SmartProxyServer} do not support shortnames in the hostnames.
 When using custom certificates, the Common Name (CN) of the custom certificate must be a fully qualified domain name (FQDN) instead of a shortname.
 This does not apply to the clients of a {Project}.
 
 ifdef::satellite[]
-* All {ProjectServer}s must be on the same version.
+* If you use multiple {ProjectServer}s, they must be on the same {Project} version for the {ISS} Export/Import to work.
```

Comment:
@parthaa Is this still true? I vaguely remember we were adding a feature to circumvent this.

Reply:
Depends on the kind of mechanism used for export/import

- If you are using an Export Sync/Air Gapped setup yes you need identical versions.
- If you are using a Network Sync on the other hand (which got added in 6.11), you should be able to sync between 6.11  and 6.12/6.13 etc.

Reply:
I'll keep the original and file another PR for this.